hudi-common/src/main/java/org/apache/hudi/metadata/MetadataPartitionType.java

```diff
@@ -22,27 +22,53 @@
 import java.util.List;
 
 public enum MetadataPartitionType {
-  FILES("files", "files-");
+  // TODO: Make the file group count configurable or auto compute based on the scaling needs
```

Comment:
Removed the file group count constant from enum. Its now set based on the config. But prefer to have the enum list for partitions so that its easy to refer the partition type in all the write and read paths.